### PR TITLE
Minor edit

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#23
#### What does this implement/fix? Explain your changes.
- Minor edit in test system

### CODE REVIEW

File: penney/__main__.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 
 if __name__ == "__main__":
-    while(True):
-        description()
-        menu()
-        control_input = input(
-            "\nPress [R] to restart or any other key to exit.")
-        if control_input.upper() != "R":
-            break
+    args = sys.argv
```
Comment (sadrasabouri) [inline penney/__main__.py:7]: Why is this for?
And shouldn't we add this change for `CHANGELOG.md`?

**Review by sadrasabouri — APPROVED** [review]
I've left a comment.
LGTM :+1:
You can merge it if my comment isn't a big deal.

File: penney/__main__.py (hunk 1 above)
Comment (sepandhaghighi) [reply penney/__main__.py:7]: It's a minor change in order to test `main` script.

**Comment by sepandhaghighi** [comment]
> I've left a comment.
> LGTM 👍
> You can merge it if my comment isn't a big deal.

If everything is okay, please merge this PR.

**Pull request merged**